doc/changelog.md

```diff
@@ -69,6 +69,11 @@
 
 <h3>Improvements</h3>
 
+* `debug.callbacks` are marked as inactive. This means `debug.callbacks` will not be considered
+  as active for the computation of gradients.
+  [(#706)](https://github.com/PennyLaneAI/catalyst/pull/706)
```

Comment:
Does this mean they can be used in a gradient call?

Reply:
Yes, I just haven't enabled them here because both `pure_callback` and `debug.callbacks` lower to the same inactive function so it doesn't make sense.

Reply:
I also don't consider this changelog finished as I plan to keep adding to it with the rest of the stories. Similar to how callbacks was documented in the changelog last release.